kcbq-confluent/src/main/java/com/wepay/kafka/connect/bigquery/schemaregistry/schemaretriever/SchemaRegistrySchemaRetriever.java

```diff
@@ -57,16 +57,16 @@ public void configure(Map<String, String> properties) {
 
   @Override
   public Schema retrieveSchema(TableId table, String topic) {
+    String subject = getSubject(topic);
     try {
-      String subject = getSubject(topic);
       logger.debug("Retrieving schema information for topic {} with subject {}", topic, subject);
       SchemaMetadata latestSchemaMetadata = schemaRegistryClient.getLatestSchemaMetadata(subject);
       org.apache.avro.Schema avroSchema = new Parser().parse(latestSchemaMetadata.getSchema());
       return avroData.toConnectSchema(avroSchema);
     } catch (IOException | RestClientException exception) {
       throw new ConnectException(
-          "Exception encountered while trying to fetch latest schema metadata from Schema Registry",
-          exception
+          "Exception while fetching latest schema metadata for topic={}, subject={}".format(
```

Comment:
This code is incorrect:

        System.err.println("Exception while fetching latest schema metadata for topic={}, subject={}".format("t","s"));

Prints "t".

You want:

        System.err.println(String.format("Exception while fetching latest schema metadata for topic=%s, subject=%s", "t", "s"));